contentcuration/search/viewsets/contentnode.py

```diff
@@ -64,23 +58,7 @@ def filter_channel_list(self, queryset, name, value):
         return queryset.filter(channel_id__in=list(channel_ids))
 
     def filter_keywords(self, queryset, name, value):
-        filter_query = Q(title__icontains=value) | Q(description__icontains=value)
-        tags_node_ids = ContentNode.tags.through.objects.filter(
-            contenttag__tag_name__icontains=value
-        ).values_list("contentnode_id", flat=True)[:250]
-        # Check if we have a Kolibri node id or ids and add them to the search if so.
-        # Add to, rather than replace, the filters so that we never misinterpret a search term as a UUID.
-        # node_ids = uuid_re.findall(value) + list(tags_node_ids)
-        node_ids = uuid_re.findall(value)
-        for node_id in node_ids:
-            # check for the major ID types
-            filter_query |= Q(node_id=node_id)
-            filter_query |= Q(content_id=node_id)
-            filter_query |= Q(id=node_id)
-        for node_id in tags_node_ids:
-            filter_query |= Q(id=node_id)
-
-        return queryset.filter(filter_query)
+        return ContentNode.search(queryset=queryset, search_term=value)
```

Comment:
Do we use this method anywhere else but here?

Reply:
Right now, we use it just here. I decoupled it keeping in mind we might need to search content nodes when we search channels.